README.md

```diff
@@ -32,6 +32,7 @@ Attributes
   Erlang.
 * `node['erlang']['install_method']` - Erlang installation method
   ("package", "source", or "esl" (for Erlang Solutions packages)).
+* `node['erlang']['use_distro_version']` - Whether to use distro version erlang packages to install.
```

Comment:
Can you adjust this to be a little clearer? Maybe something like "Use distribution-provided erlang packages"